serverless/pages/sign-up.mdx

```diff
@@ -60,7 +60,11 @@ To remove limitations, subscribe to [Elastic Cloud](https://www.elastic.co/guide
 - Third availability zone for your deployments.
 - Access to additional features, such as cross-cluster search and cross-cluster replication.
 
-You can subscribe to Elastic Cloud at any time during your trial. Billing starts when you subscribe. To maximize the benefits of your trial, subscribe at the end of the free period. To monitor charges, anticipate future costs, and adjust your usage, check your [account usage](https://www.elastic.co/guide/en/cloud/current/ec-account-usage.html) and [billing history](https://www.elastic.co/guide/en/cloud/current/ec-billing-history.html).
+You can subscribe to Elastic Cloud at any time during your trial. [Billing](/serverless/general/serverless-billing) starts when you subscribe. To maximize the benefits of your trial, subscribe at the end of the free period. To monitor charges, anticipate future costs, and adjust your usage, check your [account usage](https://www.elastic.co/guide/en/cloud/current/ec-account-usage.html) and [billing history](https://www.elastic.co/guide/en/cloud/current/ec-billing-history.html). 
+
+<DocCallOut color="warning" title="Elasticsearch minimum runtime VCUs">
+	The way that you are billed depends on the project type that you select. Some project types have a minimum number of VCUs that are always allocated and billed to your project. <DocLink slug="/serverless/general/serverless-billing" text="Learn more"/>.
```

Comment:
```suggestion
	The way that you are billed depends on the project type that you select. Some project types have a minimum number of VCUs that are always allocated to your project, incurring a minimum cost even with no active usage. <DocLink slug="/serverless/general/serverless-billing" text="Learn more"/>.
```
I was just pinged about the UI changes regarding this issue so this is timely. I think one of the pain points from customers is that we're not transparent with the costs, so let's be super explicit in addition to serving a bit of VCU jargon :D

Reply:
In the product, we're being explicit that the minimum cost is 1.25 ECU / hour. I guess we're not showing numbers in the docs, but maybe we should link more often to the website's pricing page? WDYT?

Reply:
I've gone through and applied your suggestions 🎉  - now, every time we talk about VCUs, we're linking out to the pricing page. If you have any suggestions for additional places to link to the pricing page, I'm open to them.